uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -1169,35 +1171,42 @@ RETURNS: a two element array:
 
 sub getPSC {
     my ($patientName, $dbhr) = @_;
-    my $query = "SELECT CenterID, Alias, MRI_alias FROM psc WHERE mri_alias<>''";
-    my $sth = $${dbhr}->prepare($query);
+
+    my $subjectIDsref = Settings::getSubjectIDs(
+                            $patientName,
+                            null,
+                            null,
+                            $dbhr
+                        );
+    my $PSCID = $subjectIDsref->{'PSCID'};
+    my $visitLabel = $subjectIDsref->{'visitLabel'};
+
+    ## Get the CenterID from the session table, if the PSCID and visit labels exist 
+    ## and could be extracted  
+    if ($PSCID && $visitLabel) {
+    	my $query = "SELECT s.CenterID,p.MRI_alias FROM session s 
```

Comment:
missing a space between the tow fields you select (very very nitpicky...)

Reply:
done